server/server.go

```diff
@@ -205,6 +214,10 @@ func (s *ServiceCenterServer) Stop() {
 		s.notifyService.Stop()
 	}
 
+	if s.syncerNotifyService != nil {
+		s.syncerNotifyService.Stop()
```

Comment:
what happens if syncer.enable = false

Reply:
if syncer.enable = false，  s.syncerNotifyService will be nil